examples/c/src/main.cpp

```diff
@@ -92,6 +92,9 @@ int main(int argc, char** argv) {
     return -1;
   }
 
+  // Responsible for cleaning up the library during shutdown
```

Comment:
This will impact users trying 0.2.0-rc4 with this sample.
Maybe we should wait to merge this PR? Or cherry-pick it to 0.2.0-rc5?